packages/react-adminer/src/components/TableFilter.tsx

```diff
@@ -99,8 +99,10 @@ const TableFilter: React.FC<Props> = ({ setWhere, setPage, config, where, entity
 					<Row gutter={16}>
 						{onlyStringConfigs.map(c => (
 							<Col key={c.name} span={12}>
-								{/* <Form.Item name={c.name} label={capitalCase(c.name)}> */}
-								<Form.Item name={c.name} label={c.name}>
+								<h2>
+									<Form.Item name={c.name} label={c.name} style={{ height: '0px' }} />
```

Comment:
prosim pridat i sem funkci na ziskani jmena (tu co si delal)
 ten height 0 px vypada desive, ale obcas to cssko chce nejaky magie